src/mpi/coll/scan/scan.c

```diff
@@ -153,6 +162,14 @@ int MPIR_Scan(const void *sendbuf, void *recvbuf, int count,
         mpi_errno = MPIR_Scan_impl(sendbuf, recvbuf, count, datatype, op, comm_ptr, errflag);
     }
 
+    /* Copy out data from host recv buffer to GPU buffer after reduce is complete */
```

Comment:
One minor  typo here and other similar places, it still says `reduce` .

Reply:
Fixed.